oem/ibm/configurations/bios/enum_attrs.json

```diff
@@ -451,48 +451,60 @@
             "readOnly": true
         },
         {
-            "attribute_name": "pvm_keep_and_clear",
-            "possible_values": ["Disabled", "Enabled"],
+            "attribute_name": "hb_power_PS0_functional",
```

Comment:
Why is the pvm_keep_and_clear attribute modified?

Reply:
It is not being modified,, it is in a higher part of the file. see line 410 above.
This has something to do with the tools,

Reply:
yes, looks like there were duplicate entries for the same attribute.